deploy/cr.yaml

```diff
@@ -7,6 +7,9 @@ spec:
 #  pause: false
   secretsName: cluster1-secrets
   sslSecretName: cluster1-ssl
+  toolkit:
```

Comment:
Did you ask @spron-in about this option/section ? We need to confirm all UI changes with him.

Reply:
@spron-in ping

Reply:
name looks ok to me.

I would move toolkit section to the bottom of the default cr.yaml and add commented resources subsection:
```
resources:
  requests:
    cpu:
    memory: 
  limits:
    cpu:
    memory:
```

@fiowro we will need to add info about toolkit usage to our docs. Architecture and options for sure.